### DISCUSSION TITLE
Redlight Mode

### DISCUSSION THREAD

**Discussion by LHHJohnny** [osmandapp/OsmAnd#20465] [General]
Hallo, 
Is there any possibility to activate an red light night mode, like on a submarine in stead of the general night Mode?
Maybe this option can be added within the nightmode options.
I have attached an example of my astronomical app.

Many thanks

Johnny

![20240801_092912](https://github.com/user-attachments/assets/f4d9931b-1280-462b-9685-1daec1960f64)

**Comment by sjvudp** [comment]
I'm afraid GPS will fail on a submarine, but honestly: If you set your phone to adjust the brightness automatically, you won't need the redlight mode, I guess. The only use would be when walking on foot where the road are not lit.
Comparing a navigation map with a star map is not quite fair, also (IMHO).
